### DISCUSSION TITLE
Split or clarify which settings stress the CPU and the GPU respectively.

### DISCUSSION THREAD

**Discussion by joppy-at-git** [mastercoms/tf2cfg#574] [Ideas]
I have no problem running this game GPU wise as I can run it over 144 FPS at 4K, but my FPS can still drop to two digits when a lot of CPU intensive things happens at once (even as extreme as averaging 50 FPS if you fill a room with bots). Therefor there would be nice if you could pick presets that makes everything that only demand GPU resources at high or ultra settings but keep the CPU intensive options at low or off. Or add more info in the Modules page about this.

**Comment by micwoj92** [comment]
No, this won't work.

**Comment by treacherousfiend** [comment]
While i'm not sure that presets would be a particularly good idea, as it seems like it'd add a lot more to maintain, adding details to the modules page sounds good
